graphql/schema.graphql

```diff
@@ -943,6 +943,8 @@ type Annotation {
   suspectedIssues: [IssueLink]
   createdIssues: [IssueLink]
   userCanModify: Boolean
+  # todo: make this required
```

Comment:
Why is this in a to-do is there a reason it can't be always returned?

Reply:
I'm just doing it so that I can merge this without Spruce breaking (since my front end changes aren't ready)

Reply:
That actually isn't a breaking change.

This just adds an additional field to a response that spruce does not already and does not have to request. 

It would be breaking if this was a change to an input of a query or mutation. Or if you changed the type of an existing field that spruce requests.

Reply:
good point. thanks, fixed